src/test/java/uk/ac/ebi/atlas/search/GeneSearchDaoIT.java

```diff
@@ -87,12 +91,15 @@ void validGeneIdReturnsAtLeastOneCellId(String geneId) {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"AT1G02640"})
+    @ValueSource(strings = {"AT4G11290"})
     void validGeneIdReturnsExperimentAccessions(String geneId) {
         var result = subject.fetchExperimentAccessionsWhereGeneIsMarker(geneId);
 
         assertThat(result)
-                .containsOnly("E-CURD-4");
+                .contains("E-CURD-4");
```

Comment:
Have this been reviewed? if we know how many experiments are being returned by the test can we make an assertion on number of experiments returned?

Reply:
I have mentioned a comment in the test case, have a look. we need to check with @alfonsomunozpomer why we are asserting for a particular experiment when we are getting a list.